test/correctness/vector_math.cpp

```diff
@@ -602,7 +602,7 @@ bool test(int lanes, int seed) {
     Func f22;
     f22(x, y) = absd(input(x, y), input(x+1, y));
     f22.vectorize(x, lanes);
-    Buffer<typename with_unsigned<A>::type> im22 = f22.realize(W, H);
+    Buffer<typename with_unsigned<A>::type> im22 = f22.realize(W, H); f22.invalidate_cache();
```

Comment:
Is this equivalent to making a temporary Pipeline to hold the state?
```
im22 = Pipeline(f22).realize(W, H);
```

If so, having an object represent the lifetime seems cleaner than exposing the invalidate method.

Reply:
True. (Or, alternately, wrapping the calls in {} to limit the lifespan.

Re: exposing the method, the method was *already* public on Pipeline; given that, adding to Func seems appropriate regardless.